src/Pages/AdoptAPet.tsx

```diff
@@ -183,50 +170,57 @@ const AdoptAPet = () => {
   const [isDetailsOpen, setIsDetailsOpen] = useState(false);
 
   const fetchPets = async () => {
-    try{
+    try {
       const getOperation = get({
         apiName: awsExports.aws_cloud_logic_custom[0].name,
-        path: '/pets'
-      })
+        path: '/pets',
+      });
 
       const response = await getOperation.response;
       const data: unknown = await response.body.json();
-     
-      setPets(data as Pet[])
-    }catch(e){
-      console.log(e)
+
+      setPets(data as Pet[]);
+    } catch (e) {
+      console.log(e);
     }
-  }
-  useEffect(()=>{
-    fetchPets()
-  }, [])
-  
-  const [searchAgeMin, setSearchAgeMin] = useState('');
-  const [searchAgeMax, setSearchAgeMax] = useState('');
-  const [filteredPets, setFilteredPets] = useState(pets);
-
-  const handleSearch = () => {
-    const filtered = pets.filter(
-      (pet) =>
-        (!selectedSpecies || pet.species === selectedSpecies) &&
-        (!selectedGender || pet.gender === selectedGender) &&
-        (!searchAgeMin || parseInt(pet.age) >= parseInt(searchAgeMin)) &&
-        (!searchAgeMax || parseInt(pet.age) <= parseInt(searchAgeMax))
-    );
-    setFilteredPets(filtered);
   };
   useEffect(() => {
-    handleSearch();
-  }, [selectedSpecies, selectedGender, searchAgeMin, searchAgeMax]);
+    fetchPets();
+  }, []);
+
+  const filteredPets = pets.filter(
+    (pet) =>
+      (!selectedSpecies || pet.species === selectedSpecies) &&
+      (!selectedGender || pet.gender === selectedGender)
+  );
 
   const handleAdoptClick = (pet: Pet) => {
-    setSelectedPet(pet);
+    setSelectedPet({
+      id: pet.id,
+      name: pet.name,
+      species: pet.species,
+      gender: pet.gender,
+      age: pet.age,
+      description: pet.description || 'Descripción no disponible',
+      image_url: pet.imageUrl || 'https://via.placeholder.com/250',
+      adoption_status: 'Disponible', // ⚠️ Esto es clave
```

Comment:
```suggestion
      adoptionStatus: 'Disponible', // 
```
I will work on api side to add this new column